### ISSUE TITLE
Suave Music Store

### ISSUE THREAD

**Issue by FoggyFinder** [fsharplang-ru/requests-to-translates#5]
Ссылка на оригинал - [Suave Music Store](https://www.gitbook.com/book/theimowski/suave-music-store/details)

Учитывая наличие версии на японском языке в тонкости лицензии вникать не стал

**Comment by ForNeVeR** [comment]
Я проверил там лицензию: [Apache](https://github.com/theimowski/SuaveMusicStoreTutorial/blob/master/LICENSE). Значит, никаких проблем, ссылки на оригинал будет достаточно.

**Comment by FoggyFinder** [comment]
Я думаю, что перевод будет лучше разместить вместе с оригиналом, а не тащить его в виде серии записей в блоге.

**Comment by ForNeVeR** [comment]
Да, согласен, это хорошая идея. Хостить его там же, где японский перевод.

**Comment by gsomix** [comment]
Учебник обновился.

https://twitter.com/theimowski/status/917128305974923265
